### PR TITLE
QK8S-958: Single pod related changes

### PR DESCRIPTION
**Pull request by pjaladi21**

### CODE REVIEW

File: Charts/qtest-mgr/values.yaml [hunk 1]
```diff
@@ -154,12 +154,15 @@ qTestManager:
 testconductor:
   environment:
     isOnPremise: false
+    singleInstance: false
```
Comment (liemng) [inline Charts/qtest-mgr/values.yaml:157]: We don't need single for SaaS (OD).  This is only needed for

File: Charts/qtest-mgr/values.yaml [hunk 2]
```diff
@@ -154,12 +154,15 @@ qTestManager:
...
     od:
+      single: production
```
Comment (liemng) [inline Charts/qtest-mgr/values.yaml:159]: We don't need single for SaaS (OD). This is only needed for OP.

**Comment by pjaladi21** [comment]
https://github.com/QAS-Labs/batman/blob/master/src/manager/build.gradle#L252

**Review by isgo-golgo13 — APPROVED** [review]

**Comment by pjaladi21** [comment]
Let me merge it later after verifying few things

**Comment by byangtri** [comment]
Can we also bump `version` in Chart.yaml

**Pull request merged**